docs/src/other/glossary.md

```diff
@@ -110,6 +110,12 @@ The Platform.sh CLI configures Drush aliases for you on your local environment
 (via `platform get` or `platform drush-aliases`).
 You can also configure them manually.
 
+## Environment
+An [environment](../environments/_index.md) is a standalone copy of your site,
+complete with code, data, and running services.
+Your [default branch](../environments/_index.md#default-environment) is the production environment,
```

Comment:
:hammer_and_pick: The use of both `branch` and `environment` is something that I've found confusing at times. It might help to lead with the production part?
```suggestion
Your production environment comes from the [default branch](../environments/_index.md#default-environment)
```